ansible/roles/test/tasks/ptf_runner_reboot.yml

```diff
@@ -149,5 +149,5 @@
 
 - name: Wait for the DUT to be ready for the next test
   pause: seconds=420
-  when: (preboot_list|length > 1) or
-        (inboot_list|length > 0 and 'None' not in inboot_list)
+  when: (preboot_list|length > 0 and None not in preboot_list) or
+          (inboot_list|length > 0 and None not in inboot_list)
```

Comment:
Why would None be added to the list?

Reply:
I am initializing the list with None if that list is undefined. Each item is then passed to ptf_runner